src/test/java/com/revature/services/CurriculumServiceTest.java

```diff
@@ -15,6 +15,7 @@
 import org.testng.annotations.Test;
 import com.revature.cmsforce.CMSforceApplication;
 import com.revature.entities.Curriculum;
+import com.revature.entities.CurriculumModule;
```

Comment:
why is this needed?

Reply:
i didn't work on this test, I don't know how it got added